docs/src/pages/[platform]/components/heading/HeadingPropControls.tsx

```diff
@@ -33,6 +47,12 @@ export const HeadingPropControls: HeadingPropControlsInterface = ({
         <option value="5">5</option>
         <option value="6">6</option>
       </SelectField>
+      <SwitchField
+        label="isTruncated"
+        defaultChecked={isTruncated}
```

Comment:
Same here. We should use `isChecked` to respect the practice.

Reply:
Done and changed all in docs